CefSharp/Internals/MethodParameter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace CefSharp.Internals
+{
+    [DataContract]
```

Comment:
Don't think `DataContract` is required here as we don't need to reference this in the `render process`.